### PR TITLE
chore: add sign in with id token to js docs

### PR DESCRIPTION
**Pull request by J0**
## What kind of change does this PR introduce?

Provides a basic example for [sign in with ID Token](https://github.com/supabase/gotrue-js/blob/master/src/GoTrueClient.ts#L524) w/o nonce

CleanShot 2023-08-14 at 12 16 02@2x

### CODE REVIEW

File: spec/supabase_js_v2.yml [hunk 1]
```diff
@@ -330,6 +330,20 @@ functions:
...
+          })
```
Comment (kangmingtay) [inline spec/supabase_js_v2.yml:344]: i don't think this token should be in the example

**Review by kangmingtay — CHANGES_REQUESTED** [review]

File: spec/supabase_js_v2.yml (hunk 1 above)
Comment (kiwicopple) [reply spec/supabase_js_v2.yml:344]: agreed - let's just put some filler-text

**Review by kiwicopple — APPROVED** [review]

File: spec/supabase_js_v2.yml (hunk 1 above)
Comment (J0) [reply spec/supabase_js_v2.yml:344]: Sounds good-  let’s replace.


For context, this is from the GoTrue Repo though so is alr public facing

**Comment by J0** [comment]
Re-opening since accidentally added the requirement of additional reviewers while re-basing

**Pull request closed without merging**